src/test/validate.test.js

```diff
@@ -0,0 +1,67 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../index';
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('validate middleware test', () => {
+  it('Should return error for invalid post data', (done) => {
+    const dummyUser = {
+      firstName: 'john',
+      lastName: 'doe',
+      password: 'password',
+      email: 'invalidMail'
+    };
```

Comment:
Welldone, i think firstName is meant to be first_name and lastName is meant to be last_name as that is what will be used in the user model. I got an error on postman when I used first_name and last_name as properties...

Reply:
Well the TTL said he sodent want to see any form of snake casing...

Reply:
> @oyewoas Well the TTL said he does not want to see any form of snake casing...

ok, i prefer that too